sky/templates/aws-ray.yml.j2

```diff
@@ -190,7 +190,7 @@ setup_commands:
     pip3 --version > /dev/null 2>&1 || (curl -sSL https://bootstrap.pypa.io/get-pip.py -o get-pip.py && python3 get-pip.py && echo "PATH=$HOME/.local/bin:$PATH" >> ~/.bashrc);
     (type -a python | grep -q python3) || echo 'alias python=python3' >> ~/.bashrc;
     (type -a pip | grep -q pip3) || echo 'alias pip=pip3' >> ~/.bashrc;
-    (which conda > /dev/null 2>&1 && conda init > /dev/null) || (wget -nc https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh && bash Miniconda3-latest-Linux-x86_64.sh -b && eval "$(~/miniconda3/bin/conda shell.bash hook)" && conda init && conda config --set auto_activate_base true);
```

Comment:
These couple of j2 seem to have 2 - 3 versions. (e.g., some have `conda init > /dev/null`, some don't). I guess if smoke tests all passed (does it by default cover all tests?) it's ok to use 1 version?

Reply:
It should be fine. The only difference between the two versions is just having `conda init` after the `which conda` at the beginning. Theoretically, it should not affect anything. We now have the `conda init` everywhere, and tested with `pytest tests/test_smoke.py` (on GCP) which indicates adding the `conda init` indeed does not cause problem.